Makefile.PL

```diff
@@ -0,0 +1,10 @@
+use strict;
```

Comment:
I think we can probably get rid of this.

Reply:
The cargo cult strikes again.